src/event.c

```diff
@@ -0,0 +1,591 @@
+
+// TODO: actual events
+
+extern const u16 gEvent_DisplayBattleQuote[]; /*
+    1020 0003 | EVBIT_MODIFY 3  // modifies event state bits (how?)
+    1B20 FFFF | TEXTSHOW 0xFFFF // Show text in event slot 2
+    1D20 0000 | TEXTEND         // Ends text
+    1B22 0000 | REMA            // deletes all text-related procs (or something)
+    0120 0000 | ENDA            // ends event execution
+*/
+
+extern const u16 gEvent_TriggerQueuedTileChanges[]; /*
+    1020 0001 | EVBIT_MODIFY 1    // modifies event state bits (how?)
+    2720 FFFD | TILECHANGE 0xFFFD // Changes tile by tile change id in event slot Queue (or something)
+    0120 0000 | ENDA              // ends event execution
+*/
+
+extern const u16 gEvent_OpenChest[]; /*
+    1020 0001           | EVBIT_MODIFY 1
+    2720 FFFD           | TILECHANGE 0xFFFD
+    0540 0007 000000FF  | SVAL s7 0xFF
+    0C43 0000 0003 0007 | BGT 0 s3 s7
+    3720 FFFF           | GIVEITEMTO 0xFFFF
+    0920 0001           | GOTO 1
+    0820 0000           | LABEL 0
+    3721 FFFF           | GIVEITEMTOMAIN 0xFFFF
+    0820 0001           | LABEL 1
+    0120 0000           | ENDA
+*/
+
+extern const u16 gEvent_MapSupportConversation[]; /*
+    1020 0003           | EVBIT_MODIFY 3
+    0C40 0000 0002 0000 | BEQ 0 s2 s0
+    1220 FFFF           | MUSC 0xFFFF
+    0920 0001           | GOTO 1
+    0820 0000           | LABEL 0
+    1520 0000           | MUSI
+    0820 0001           | LABEL 1
+    0620 0032           | SADD $032
+    1B20 FFFF           | TEXTSHOW 0xFFFF
+    1D20 0000           | TEXTEND
+    1B22 0000           | REMA
+    3A40 000C 005A 0000 | NOTIFY 0xC 0x5A 0
+    0228 0007           | EVBIT_T 7 // NoFade
+    0120 0000           | ENDA
+*/
+
+extern const u16 gEvent_SupportViewerConversation[]; /*
+    1020 0003           | EVBIT_MODIFY 3
+    1A21 0000           | REMOVEPORTRAITS
+    2140 0037 0000 0000 | BACG 0x37
+    1720 0010           | FADU 16
+    1B20 FFFF           | TEXTSHOW 0xFFFF
+    1D20 0000           | TEXTEND
+    1B22 0000           | REMA
+    1721 0010           | FADI 16
+    0120 0000           | ENDA
+*/
+
+extern const u16 gEvent_SkirmishRetreat[]; /*
+    EVBIT_MODIFY 4
+    TUTORIALTEXTBOXSTART
+    SVAL sB (-1)
+    TEXTSHOW 0x8FC // "Retreat?        [NL][No][X]"
+    TEXTEND
+    SVAL s7 TRUE
+    BNE 0 sC s7
+    MUSCMID 0x7FFF
+    FADI 4
+    MNCH 0xFFFF
+    CHECK_SKIRMISH
+    SVAL s1 1
+    BNE 0 sC s1
+    ASMC 0x37D59
+    LABEL 0
+    REMA
+    EVBIT_T 7 // NoFade
+    ENDA
+*/
+
+extern const u16 gEvent_SuspendPrompt[]; /*
+    EVBIT_MODIFY 4
+    TEXTSTART
+    TEXTSHOW 0x8FE
+    TEXTEND
+    SVAL s7 TRUE
+    BNE 0 sC s7
+    ASMC 0xB5D5D // Save Suspended game
+    TEXTSHOW2 0x8FF
+    TEXTEND
+    MUSCMID 0x7FFF
+    FADI 4
+    MNTS 0
+    LABEL 0
+    REMA
+    EVBIT_T 7
+    ENDA
+*/
+
+extern const u16 gEvent_GameOver[]; /* Game Over Events?
+    1020 0004           | EVBIT_MODIFY 4
+    0D40 0000 08085375  | ASMC 0x8085375 (game over)
```

Comment:
identify the function and label/use it instead of a raw addr, even in the comments